doc/source/development/extending.rst

```diff
@@ -499,4 +501,4 @@ registers the default "matplotlib" backend as follows.
 
 
 More information on how to implement a third-party plotting backend can be found at
-https://github.com/pandas-dev/pandas/blob/master/pandas/plotting/__init__.py#L1.
\ No newline at end of file
+https://github.com/pandas-dev/pandas/blob/master/pandas/plotting/__init__.py#L1.
```

Comment:
Was this a whitespace change?

Reply:
Seems like it because the change didn't show up in `git diff -w`. But I still can't see the exact change. Should I let it slide?

Reply:
Must have been EOF newline, it's fine